Jenkinsfile

```diff
@@ -0,0 +1,60 @@
+pipeline {
```

Comment:
Where this pipeline is running? Can we have access?
The github solutions like travis are enable on this project?